docs/index.md

```diff
@@ -459,11 +459,11 @@ Also, it will include the following headers:
 
 There are very few limitations imposed on object names. Object names must
 contain only valid UTF-8 characters and may be a maximum of 1024 characters in
-length. Object names may include forward slash characters (or any other valid
-UTF-8 character) to create the suggestion of a directory hierarchy for a set of
-object even though the buckets system uses a flat namespace. Care must be
-taken, however, to properly URL encode all object names to avoid problems when
-interacting with the server.
+length (in bytes). Object names may include forward slash characters (or any
```

Comment:
s/1024 characters in length (in bytes)/1024 bytes/   or similar.

Reply:
will do